doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,272 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing circuits
+==================
+
+PennyLane offers the :mod:`~.io` module to import quantum circuits and operations that were
+constructed outside of PennyLane. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
```

Comment:
```suggestion
constructed in another library. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
```
Just to avoid repetition of saying PL twice.

Reply:
That's fair although I don't think "library" is the best choice since OpenQASM (and Quil) aren't exactly libraries. Can you think of a more general way to rephrase this one? Perhaps
```suggestion
constructed using another framework. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
```